### DISCUSSION TITLE
Will there be tools for encrypting game content in axmol?

### DISCUSSION THREAD

**Discussion by 10100010010001001010** [axmolengine/axmol#1818] [General]
Will there be tools for encrypting game content in axmol? I would like to prevent the user from seeing sprites and other file types after compilation.

**Comment by rh101** [comment]
[Texture Packer](https://www.codeandweb.com/texturepacker) supports exporting the encrypted format (`.pvr.ccz`), and while it helps to protect the game assets, it won't prevent a knowledgeable person from extracting those assets.  You can only slow down the process, but there is no way to prevent it from happening.

**Reply by 10100010010001001010** [reply]
yeah, I know it's impossible to completely prevent this, but I would like to at least hide them from direct viewing/editing. Thanks for help
